modules/variance.js

```diff
@@ -0,0 +1,22 @@
+import cb from './_cb.js';
+import mean from './mean.js';
+
+// Return the variance of the numeric elements of the collection,
+// optionally after transforming them through `iteratee`.
+// https://en.wikipedia.org/wiki/Variance
+export default function variance(collection, iteratee, context) {
+  if (typeof iteratee == 'number' && collection != null && typeof collection[0] != 'object') iteratee = null;
```

Comment:
Let's not make lines longer than necessary.

```suggestion
  if (typeof iteratee == 'number' && collection != null && typeof collection[0] != 'object') {
    iteratee = null;
  }
```

or

```suggestion
  if (
    typeof iteratee == 'number' && collection != null && typeof collection[0] != 'object'
  ) iteratee = null;
```